docs/conf.py

```diff
@@ -44,7 +44,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+html_theme = 'nature'
```

Comment:
Just out of curiosity, was there any particular reason to switch to 'nature'? Alabaster looked nice.

Reply:
I wanted more color.